### ISSUE TITLE
SchemaView without_ui/with_ui return a function, not ApiView

### ISSUE THREAD

**Issue by palfrey** [intgr/drf-yasg-stubs#2]
In both cases they've called [as_cached_view](https://github.com/axnsan12/drf-yasg/blob/9a89d8ccb0b6966a3f5549b8f4f46d08b6e8d854/src/drf_yasg/views.py#L111) which calls `as_view`

With the current stubs I get
```
error: No overload variant of "url" matches argument types "str", "APIView", "str"
note: Possible overload variants:
note:     def url(regex: str, view: Callable[..., HttpResponseBase], kwargs: Dict[str, Any] = ..., name: str = ...) -> URLPattern
note:     def url(regex: str, view: Tuple[List[URLResolver], Optional[str], Optional[str]], kwargs: Dict[str, Any] = ..., name: str = ...) -> URLResolver
note:     def url(regex: str, view: List[Union[URLResolver, str]], kwargs: Dict[str, Any] = ..., name: str = ...) -> URLResolver
```

They should be marked as returning some sort of Callable variant I think

**Comment by intgr** [comment]
Thanks, right you are. Sorry it took me so long to get around to this.

I will publish a fix soon.

**Comment by intgr** [comment]
Closed in #4.

**Comment by intgr** [comment]
Uploaded version 0.1.1 with this fix and a few others.
